map/place_page_info.cpp

```diff
@@ -201,6 +202,12 @@ std::string Info::FormatSubtitle(bool withType) const
   if (!fee.empty())
     append(fee);
 
+  // Debug types
+  bool debugAllTypesSetting = false;
+  settings::TryGet(kDebugAllTypesSetting, debugAllTypesSetting);
+  if (debugAllTypesSetting)
+    append(GetAllReadableTypes());
```

Comment:
Are all types appended to "filtered" GetAllLocalizedTypes ? Does it look good?

What's the goal?

Reply:
I think it's useful when debugging some things to be able to see all the internal types. It's not for users to use, it's mainly for developers. When this debug command is activated, all types are appended at the end of the description in the place page. See https://github.com/organicmaps/organicmaps/pull/8481#issuecomment-2195167310

Reply:
If you guys think it's not useful, I can drop the commit. No problem.

Reply:
We are using the desktop version for debugging and development, it shows all raw tags. Also, repeating the same but not translated tags doesn't look convenient to read/debug.

Do you use the desktop version?

Reply:
Its a useful feature, sometimes its very helpful to debug on a device.
Also power users were asking for it several times.

Reply:
At the moment, all useful types should be printed. In general, debugging on a device is an interesting idea, it can be implemented with `#ifdef DEBUG` to avoid unnecessary overhead in production.

Users were asking for it because they miss/expect to see standard osm tags. It's not the case.

What are your use-cases for debugging on the device?

Reply:
E.g. road attribs like surface, hwtags - power users will be able to understand why routing avoids or prefers certain ways.
It'd be possible to check many things on the go without desktop access.
`ifdef` is not needed - PP is not opened many times a second, there won't be overhead. Actually restricting it to debug version only will kill 90% of convenience and benefits of this feature.

Reply:
Road surface should be visible to all users without any debug commands.